src/kepler_model/server/model_server.py

```diff
@@ -205,18 +206,25 @@ def get_available_models():
         if energy_source is None or "rapl" in energy_source:
             energy_source = "rapl-sysfs"
 
+        if node_type is None:
+            node_type = any_node_type
+        else:
+            node_type = int(node_type)
```

Comment:
can we also add type hints to `select_best_model` to indicate that `node_type` must be `int` ?